09-features.md

```diff
@@ -41,6 +41,7 @@ The Context column decodes as follows:
 | 20/21 | `option_anchor_outputs`          | Anchor outputs                                            | IN       | `option_static_remotekey` | [BOLT #3](03-transactions.md)         |
 | 22/23 | `option_anchors_zero_fee_htlc_tx` | Anchor commitment type with zero fee HTLC transactions   | IN       |                   | [BOLT #3][bolt03-htlc-tx], [lightning-dev][ml-sighash-single-harmful]|
 | 26/27 | `option_shutdown_anysegwit`         | Future segwit versions allowed in `shutdown`              | IN       |                   | [BOLT #2][bolt02-shutdown]   |
+| 38/39 | `option_onion_messages`        | Can forward onion messages                                | IN9        |                   | [BOLT #7](07-routing-gossip.md#onion-messages) |
```

Comment:
Why is the forwarding capability defined as an invoice feature?

Reply:
I guess it's the description that's incorrect, it should be a more general "Supports onion messages"? You need to put that into the invoice so that senders know they can send you onion messages if they don't have your `node_announcement`.

Reply:
If you accept onion messages, but don't forward them, how is this communicated? Only flag the bit in the invoice and not in the node announcement?

Reply:
> Only flag the bit in the invoice and not in the node announcement?

Yes that could indeed be a good way of communicating this!
Do you think it's insufficient?

Reply:
Actually, is there even a reason for advertising the capability in the invoice at all? For bolt12, the messaging is already completed when you've got the invoice.

Reply:
Well, if you give someone a bolt12, you're kind of implying you can talk to them.  I agree with Joost, it's a weird thing to put in an invoice at all.